src/udev/udev-rules-util.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "escape.h"
+#include "macro.h"
+#include "utf8.h"
+
+static inline int parse_value(char *str, char **ret_value, char **ret_endpos) {
```

Comment:
Please do not introduce such a long function in a header. Please move this to e.g. `src/libudev/libudev-util.c`.

Reply:
Is dropping `static` ok here? I'm not sure how to unit test it otherwise.

Reply:
Or src/shared/udev-util.c may be also OK.